src/content/fr/2022/jamstack.md

```diff
@@ -0,0 +1,442 @@
+---
+#See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
+title: Jamstack
+description: Le chapitre Jamstack de l'Almanac du Web 2022 couvre la quantification des sites Jamstack, la croissance de Jamstack, les frameworks de Jamstack et l'hébergement.
+authors: [seldo, whitep4nth3r]
+reviewers: [tunetheweb]
+analysts: [seldo, tunetheweb]
+editors: [DesignrKnight]
+translators: []
```

Comment:
```suggestion
translators: [Xav83]
```

Could you also add yourself to `src/config/2022.json near the bottom in Alphabetical order by first name?

Something like this, just before ydimova:

```json
    "Xav83":{
      "name": "Xavier Jouvenot",
      "teams":[
        "translators"
      ],
      "avatar_url": "6787157",
      "website": "https://10xlearner.com/",
      "github": "Xav83",
      "linkedin": "xavier-jouvenot-98787794",
      "twitter": "10xLearner"
    }
```

And actually if you wouldn't mind moving harendra and joeviggiano that I see are not in alphabetical order while you're at it, that would be greatly appreciated! 🙏

Reply:
Hi @tunetheweb, I have moved `harendra` and `joeviggiano` as you request, but I have seen several others that are not in the alphabetical order, it may be useful to do a dedicated Pull Request to reorder the elements in that file :wink:

Reply:
Yeah let's merge what we have and I'll do a separate PR for those. Not essential they are in alphabetical order, but makes it easier to find people when looking at the file manually. Thanks for cleaning those two up anyway!